src/util/zod/order.schema.ts

```diff
@@ -2,8 +2,9 @@ import { z } from 'zod';
 // import { PaymentMethodEnum, PaymentStatusEnum } from '../../type/ticket.type';
 import { validateInt } from '../validator';
 
-export const userAttendSchema = z.object({
-  userInfo: z.object({
+export const createOrderSchema = z.object({
+  ticketId: z.string({ required_error: 'Ticket id is required' }),
+  orderContact: z.object({
```

Comment:
payment.method in model is required but not default value, so here should required method argument